core/templates/dev/head/app.js

```diff
@@ -102,6 +102,8 @@ oppia.constant('DEFAULT_PROFILE_IMAGE_PATH', '/avatar/user_blue_72px.png');
 oppia.constant('FEEDBACK_POPOVER_PATH',
   '/pages/exploration_player/feedback_popup_container_directive.html');
 
+oppia.constant('LOGOUT_REDIRECT_URL', '/logout');
```

Comment:
Seems like LOGOUT_URL to me.... no more redirect?

Reply:
Done.